docs/user_guide/mitigation/reductions.rst

```diff
@@ -601,7 +601,7 @@ Here is an example of how to instantiate an :class:`ExponentiatedGradient` model
     >>> # Create a pipeline with the preprocessor and estimator
     >>> estimator = Pipeline([
     ...     ('preprocessor', preprocessor),
-    ...     ('classifier', RandomForestClassifier(n_estimators=10, random_state=42))])
+    ...     ('classifier', HistGradientBoostingClassifier(n_estimators=10, random_state=42))])
```

Comment:
```suggestion
    ...     ('classifier', HistGradientBoostingClassifier(random_state=42))])
```

Reply:
Hi @milenaarneschedle, `HistGradientBoostingClassifier` doesn't have a `n_estimators=10` parameter as the `RandomForestClassifier` does: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html.